installer/Migration/V5_4_0/lang-string/auth.yaml

```
@@ -53,3 +53,6 @@ langStrings:
   -
     value: 'This request is invalid'
     unitId: this_request_is_invalid
+  -
+    value: 'Please enable the Mobile option in the module configuration screen'
```

I'll double check this message

---

```suggestion
    value: 'Mobile app is not enabled. Please contact your OrangeHRM administrator'
```
Since enabling the mobile app is an admin-level feature, ESS users who see this message may find it confusing as they do not have the necessary permissions to enable the app themselves.